package.json

```diff
@@ -37,7 +37,8 @@
     "linklocal": "^2.8.1",
     "lint-staged": "^4.1.3",
     "lodash": "^4.17.4",
-    "material-ui": "next",
+    "material-ui": "^1.0.0-beta.11",
+    "material-ui-prev": "npm:material-ui@latest",
```

Comment:
what is this?

Reply:
juts that, my attempt to use both versions. The prev one is not needed any more